tools/docker/compose

```diff
@@ -38,6 +38,10 @@ Commands:
     integration | i       Run docker-compose with config for integration tests
     dev:integration | di  Run docker-compose with dev config for integration tests
     dev                   Run docker-compose with dev config for the core node and operator-ui
+
+    eth:clean             Reset blockchain data to genesis state
+    node:clean            Reset chainlink database
```

Comment:
What do you think about naming this to `cl:clean`? Seems less ambiguous than `node`. `node` is a little overloaded in meaning

Reply:
yeah agreed!